src/global_scheduler/global_scheduler.h

```diff
@@ -39,10 +39,12 @@ typedef struct {
   UT_hash_handle hh;
 } scheduler_object_info;
 
+/* plasma_manager ip:port -> photon_db_client_id association entry. */
```

Comment:
Can we put the documentation into the same form as everywhere else, i.e. full sentences, doxygen style (i.e. /** instead of /*) and for members preceding the member and not following it.